kani-driver/src/cbmc_output_parser.rs

```diff
@@ -532,6 +570,8 @@ pub fn process_cbmc_output(
         // TODO: Record processed items and dump them into a JSON file
         // <https://github.com/model-checking/kani/issues/942>
     }
+
+    parser.flush_buf_writer();
```

Comment:
This could be called in the iterator instead (case where `len == 0`). Not too important, but maybe you want to keep all buffer writing operations there for clarity.

Reply:
Yup, that's cleaner. Done